Postman Test Suite/Ed-Fi ODS-API Integration Test Suite ResponseTests.postman_collection.json

```diff
@@ -5646,33 +5699,21 @@
 													"});",
 													"",
 													"pm.test(\"Should indicate that the classPeriodReferences in the child collection are inconsistent with the parent context.\", () => {",
-													"  pm.expect(problemDetails.validationErrors).to.deep.equal({",
-													"    \"$.classPeriods[0].classPeriodReference.schoolId\": [",
-													"      \"All values supplied for 'schoolId' must match. Review all references (including those higher up in the resource's data) and align the following conflicting values: '777', '888'\"",
-													"    ],",
-													"    \"$.classPeriods[1].classPeriodReference.schoolId\": [",
-													"      \"All values supplied for 'schoolId' must match. Review all references (including those higher up in the resource's data) and align the following conflicting values: '777', '999'\"",
-													"    ],",
-													"    \"$.classPeriods[2].classPeriodReference.schoolId\": [",
-													"      \"All values supplied for 'schoolId' must match. Review all references (including those higher up in the resource's data) and align the following conflicting values: '777', '999'\"",
-													"    ],",
-													"    \"$.classPeriods[3].classPeriodReference.schoolId\": [",
-													"      \"All values supplied for 'schoolId' must match. Review all references (including those higher up in the resource's data) and align the following conflicting values: '777', '999'\"",
-													"    ],",
-													"    \"$.classPeriods[4].classPeriodReference.schoolId\": [",
-													"      \"All values supplied for 'schoolId' must match. Review all references (including those higher up in the resource's data) and align the following conflicting values: '777', '999'\"",
-													"    ],",
-													"    \"$.classPeriods[5].classPeriodReference.schoolId\": [",
-													"      \"All values supplied for 'schoolId' must match. Review all references (including those higher up in the resource's data) and align the following conflicting values: '777', '999'\"",
-													"    ],",
-													"    \"$.classPeriods[6].classPeriodReference.schoolId\": [",
-													"      \"All values supplied for 'schoolId' must match. Review all references (including those higher up in the resource's data) and align the following conflicting values: '777', '999'\"",
-													"    ]",
-													"  });",
+													"  // We should see at least 6 validation messages (for each member of the child collection)",
+													"  // but may also produce one for: \"$.schoolReference.schoolId\"",
```

Comment:
Producing a message for `$.schoolReference.schoolId` would not be considered incorrect (even though current Ed-Fi ODS API does not produce this one).